tests/js/test-caseholder.js

```diff
@@ -0,0 +1,495 @@
+        wildcardDeepRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/wildcard/many/levels/deep"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        cookieSetRequest: {
+            type: "kettle.test.request.httpCookie",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/cookie"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        cookieReadRequest: {
+            type: "kettle.test.request.httpCookie",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/reqview"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        sessionRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/reqview"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        bodyParserRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: {
+                    expander: {
+                        funcName: "gpii.express.tests.expressTestCaseHolder.assembleUrl",
+                        args:     ["{testEnvironment}.options.baseUrl", "/reqview"]
+                    }
+                },
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        },
+        shutdownRequest: {
+            type: "kettle.test.request.http",
+            options: {
+                path: "{testEnvironment}.options.baseUrl",
+                port: "{testEnvironment}.options.port",
+                method: "GET"
+            }
+        }
+    },
+    modules: [
+        {
+            tests: [
+                {
+                    name: "Testing the 'static' router module (index content)...",
+                    type: "test",
+                    sequence: [
```

Comment:
We may need to review this together.  When I tried to do this, I started getting errors about not being able to figure out `{arguments}.0`:

` FATAL ERROR: Uncaught exception: Failed to resolve reference {arguments}.0 - could not match context with name arguments from component `

I am committing my work in progress so that we can review together next week.